src/app/components/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -1,22 +1,19 @@
-<div class="”modal-header”">
```

Comment:
first of all about the title : finished* editing - you can't have Two progressive tenses in a row - meaning I cant say: finishing editing